File: solved.ac/CLASS2/11866.py
import sys
input = sys.stdin.readline

n, k = map(int, input().split())
arr = [i+1 for i in range(n)]
idx = 0
answer = []
while arr:
    idx += k-1
    idx %= len(arr)
    answer.append(arr.pop(idx))
if n == 1:
    print("<" + str(answer[0]) + ">")
else:
    for i in range(len(answer)):
        if i == 0:
            print("<" + str(answer[i]) +",", end = " ")
        elif i == len(answer)-1:
            print(str(answer[i]) + ">")
        else:
            print(str(answer[i])+",", end = " ")

File: solved.ac/CLASS2_JS/11866.js
const fs = require("fs");
const inputPath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const stdin = fs
  .readFileSync(inputPath)
  .toString()
  .split("\n")
  .map((line) => line.trim());

const input = (() => {
  let line = 0;
  return () => stdin[line++];
})();

const solution = () => {
  const [n, k] = input().split(" ").map(Number);
  const arr = Array.from({ length: n }, (_, index) => index + 1);
  const ans = [];
  while (arr.length) {
    for (let i = 0; i < k - 1; i++) {
      arr.push(arr.shift());
    }
    ans.push(arr.shift());
  }

  console.log(`<${ans.join(", ")}>`);
};

solution();
